boards/arm/arduino_due/board.h

```diff
@@ -7,6 +7,8 @@
 #ifndef __INC_BOARD_H
 #define __INC_BOARD_H
 
+#ifndef _ASMLANGUAGE
```

Comment:
this should probably be done in soc.h, not here.

(should probably also send a patch to fixup SAME70 to do the same thing